mmv1/products/firebase/api.yaml

```diff
@@ -57,16 +57,77 @@ objects:
+          are intended solely for developers and client-side tools Firebase services
+          will not mutate this annotation set.  This field may only be assigned on Update.
+      - !ruby/object:Api::Type::NestedObject
+        name: resources
+        output: true
+        description: |
+          The default Firebase resources associated with the Project.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: hostingSite
+            output: true
+            description: |
+              The default Firebase Hosting site name, in the format:
+              `PROJECT_ID` Though rare, your `projectId` might already be used as
+              the name for an existing Hosting site in another project (learn more
+              about creating non-default, [additional sites](https://firebase.google.com/docs/hosting/multisites)).
+              In these cases, your `projectId` is appended with a hyphen then five
+              alphanumeric characters to create your default Hosting site name.
+              For example, if your `projectId` is `myproject123`, your default Hosting
+              site name might be: `myproject123-a5c16`.
+          - !ruby/object:Api::Type::String
+            name: locationId
+            output: true
+            description: |
+              The ID of the Project's default GCP resource location.
+              The location is one of the available [GCP resource locations](https://firebase.google.com/docs/projects/locations).
+              This field is omitted if the default GCP resource location has not
+              been finalized yet. To set a Project's default GCP resource location,
+              call [`FinalizeDefaultLocation`](../projects.defaultLocation/finalize)
+              after you add Firebase resources to the Project.
+          - !ruby/object:Api::Type::String
+            name: realtimeDatabaseInstance
+            output: true
+            description: |
+              The default Firebase Realtime Database instance name,
+              in the format: `PROJECT_ID` Though rare, your `projectId` might already
+              be used as the name for an existing Realtime Database instance in
+              another project (learn more about [database sharding](https://firebase.google.com/docs/database/usage/sharding)).
+              In these cases, your `projectId` is appended with a hyphen then five
+              alphanumeric characters to create your default Realtime Database instance
+              name. For example, if your `projectId` is `myproject123`, your default
+              database instance name might be: `myproject123-a5c16`
+          - !ruby/object:Api::Type::String
+            name: storageBucket
+            output: true
+            description: |
+              The default Cloud Storage for Firebase storage bucket, in the format: `PROJECT_ID.appspot.com`.
   - !ruby/object:Api::Resource
     name: 'ProjectLocation'
```

Comment:
project location is handled its own resource right now: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/firebase_project_location

Reply:
Are you suggesting to remove it from here?

Reply:
probably.  it's nearly an obsolete concept, and it would create redundant ways to interact with it.  

If we omit it, we can just retire/deprecate the old project_location resource later when it is no longer meaningful.

Reply:
I will do this in a separate PR.